src/test/java/utils/TestContext.java

```diff
@@ -0,0 +1,24 @@
+package utils;
+
+import pojo.Booking;
+
+public class TestContext {
+    private static int bookingid;
```

Comment:
please use Lombok @Getter, @Setter on static var level, instead of implementing setters/getters